test-tests/shared/src/test/scala/zio/test/SpecSpec.scala

```diff
@@ -199,6 +245,24 @@ object SpecSpec extends ZIOBaseSpec {
           }
         ).provideSomeLayerShared[Scope](layer).provideLayer(Scope.default) @@ sequential
         assertZIO(succeeded(spec))(isTrue)
+      },
+      test("propagates the scope to multiple tests") {
```

Comment:
This title is reused, which in the past I found that it doesn't play very nice in some cases. Also in zio-test, it's generally more idiomatic to structure tests hierarchically, e.g.,

```scala
suite("propagates the scope to multiple tests")(
  test("provideLayerShared") { ... },
  test("provideSomeLayerShared") { ... }
)
```

Reply:
Oops, my bad, I just realised they're already scoped. Please disregard my comment